src/MusicStore/Startup.cs

```diff
@@ -81,12 +83,15 @@ public void ConfigureServices(IServiceCollection services)
             // Add the system clock service
             services.AddSingleton<ISystemClock, SystemClock>();
 
+            services.AddSingleton<ILogger>((s) => { return new ConsoleLogger("main", null, false); });
```

Comment:
Need not do this. There is already a service which resolves `ILogger<>`
https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/LoggingServiceCollectionExtensions.cs#L28